bin/erblint

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
```

Comment:
The pattern now is to public facing executables of gems being in the `exe` directory.